### PR TITLE
Add public activity to `RawPendingStripeTransaction`

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

**Comment by garyhtou** [comment]
I'm not sure if we want this at the "Raw" level. i think it might belong higher in the transaction engine

**Comment by sampoder** [comment]
I initially thought that as well and was trying to find a better place, however, this felt like the only place where on create we could reliably get a event and a user. Any alternatives you'd prefer?

File: app/views/public_activity/raw_pending_stripe_transaction/_create.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<%= render layout: "/public_activity/common", locals: { activity:, icon: "checkmark", color: "success" } do %>
+  spent <%= render_money activity.trackable.canonical_pending_transaction.local_hcb_code.amount_cents.abs %> on <i><%= render_money activity.trackable.canonical_pending_transaction.local_hcb_code.memo %></i>
```
Comment (garyhtou) [inline app/views/public_activity/raw_pending_stripe_transaction/_create.html.erb]: let's make sure this handles refunds

**Review by garyhtou — COMMENTED** [review]
🪄

**Pull request merged**